local-store.js

```diff
@@ -55,6 +59,20 @@ LocalStore.prototype = {
     this.serialize()
   },
 
+  getLastAdded: function getLastAdded () {
```

Comment:
This methods should be created automatically by https://github.com/logux/logux-client/blob/f477f962749731c701de85b6091a6f918b5fcbab/local-store.js#L37